src/metabase/collections/api.clj

```diff
@@ -1003,7 +998,7 @@
   Note that this endpoint should return results in a similar shape to `/api/dashboard/:id/items`, so if this is
   changed, that should too."
   [{:keys [id]} :- [:map
-                    [:id ms/PositiveInt]]
+                    [:id [:or ms/PositiveInt :string]]]
```

Comment:
`[:or ms/PositiveInt :string]` makes it accept strings (i.e. entity ids); not sure if I should add an `EntityId` schema type instead.